### ISSUE TITLE
Required Content

### ISSUE THREAD

**Issue by thejoefriel** [yalla-coop/presspad#55]
Content required from PressPad
- [ ] Total list of questions to ask intern
- [ ] incl. list of absolutely required questions for them to fill in when they first sign up
- [ ] What answers/fields of intern are shown publicly to hosts
- [ ] Wordings for emails:
- [ ] Sent to host and intern when booking is confirmed
- [ ] Sent to host and intern to remind them about things left to do
- [ ] Videos: 
- [ ] landing page video
- [ ] video explaining why donations are important
- [ ] List of things the intern needs to finish before their stay for the checklist
- [ ] List of things the host needs to finish before their stay for the checklist

**Comment by RamyAlshurafa** [comment]
- [ ] social media links for the footer ( Twitter, Facebook, Youtube, Linkedin, Instagram)

**Comment by thejoefriel** [comment]
Hi @OliviaPressPad @laurapresspad - we've flagged in this issue some content / assets we need from you please :)

**Comment by laurapresspad** [comment]
Hi all,

When do you need the content for the website by? Give me a deadline and we´ll get it done. In the meantime here are our social links.

- https://www.facebook.com/PressPadUK/
- https://www.instagram.com/presspaduk/
- https://twitter.com/PressPAduk
- https://www.linkedin.com/company/press-pad/
- https://www.youtube.com/channel/UCfRP3mFgrMWM6_T98I2qVNA

**Comment by thejoefriel** [comment]
Thanks @laurapresspad - would end of next week be ok? Or if that's too tight, end of the following week would be fine too as we can use placeholders for now

**Comment by thejoefriel** [comment]
Hi @OliviaPressPad @laurapresspad - really sorry to chase but we still haven't received any of the content we still need. It's really important we get this so we can properly build out the pages. Would you be able to send this this week please?

**Comment by OliviaPressPad** [comment]
Hi Joe,
Of course. We didn’t get a deadline so wasn’t sure when you needed it by. We’ll get it to you by Friday.
Thanks
Olivia

**Comment by thejoefriel** [comment]
Hey @OliviaPressPad - ah sorry! Yeah if you could get it by Friday that would be amazing! thank you! 

I'm also going to create a separate issue on the checklist bit for bookings, as this is clearly a super important bit to ensure you have full oversight and I think we need more thinking as to how this should work and look.

**Comment by thejoefriel** [comment]
Hi @OliviaPressPad - just checking in to see how you're going with this? :)

**Comment by thejoefriel** [comment]
Hey @OliviaPressPad @laurapresspad - let me know if there's any issues getting this across. We could really do with it as it's holding up the speed we can work on the platform currently. Thanks!

**Comment by thejoefriel** [comment]
Hey @OliviaPressPad - I saw you shared a document with us but it doesn't have any of the information in there currently. Is this still a working progress?

**Comment by thejoefriel** [comment]
@FarahZaqout @ashatat @RamyAlshurafa @amusameh - here's a link to a document Olivia and the team has been filling in covering the required content: https://docs.google.com/document/d/1IOql3pSVmnQYbIlOtCmTUMYAHtoAQrBpOYQnQeHbRlI/edit?usp=sharing

You can see I've raised a few questions. Please feel free to add any others you feel I've missed.

**Comment by thejoefriel** [comment]
Going to close this now as we've since moved on and best to have a clean issue for future sprints
